README.md

```diff
@@ -74,6 +76,36 @@ When you make a payment in a currency other than XRP, you also need to include t
 
 Although the Ripple-REST API provides a high-level interface to Ripple, there are also API methods for checking the status of the `rippled` server and retrieving a Ripple transaction in its native format.
 
+### Transaction Flow ###
+
+First you submit a transaction by choosing the three-step, two-step, or one-step submission methods, then you verify the transaction. The steps are described below.
+
+#### Transaction Endpoints ####
+
+* [Submit Payment - `POST /v1/accounts/{:source_address}/payments`](#submit-payment)
+* [Place Order - `POST /v1/accounts/{:address}/orders`](#place-order)
+* [Cancel Order - `DELETE /v1/accounts/{:address}/orders/{:sequence}`](#cancel-order)
+* [Grant Trustline - `POST /v1/accounts/{:address}/trustlines`](#grant-trustline)
+* [Update Account Settings - `POST /v1/accounts/{:address}/settings`](#update-account-settings)
+
+#### Three-step Submission ####
+1. Create a transaction with one of the transaction endpoints using the `submit=false` query parameter and omit the `secret` in the request.
+2. Sign the transaction by sending the `tx_json` from the response of step 1 and the `secret` to the [`sign`](#sign-transaction) endpoint.
+  * Persist the transaction `hash` before step 3.
+3. Submit the transaction by sending the `tx_blob` from the response of step 2 to the [`submit`](#submit-transaction) endpoint.
+
+#### Two-step Submission ####
+1. Create a transaction with one of the transaction endpoints using the `submit=false` query parameter and include the `secret` in the request.
+  * Persist the transaction `hash` before step 2.
```

Comment:
Not really because you have to verify the transaction for any practical use case.